src/wallet/KintoWalletFactory.sol

```diff
@@ -204,6 +204,12 @@ contract KintoWalletFactory is Initializable, UUPSUpgradeable, OwnableUpgradeabl
         IFaucet(_faucet).claimKintoETH(_signatureData);
     }
 
+    function sendMoneyToAccount(address target) external payable override {
+        require(owner() == msg.sender || kintoID.isKYC(msg.sender), "KYC required");
```

Comment:
we can use the `onlyOwner` modifier and for the `kintoID.isKYC(msg.sender)` requirement, we could do a new `modifier onlyKYCd` which can be used on the other parts having this same requirement